src/app/core-ui/main/status/status.component.ts

```diff
@@ -91,4 +93,9 @@ export class StatusComponent implements OnInit {
         break;
     }
   }
+
+  /* Open Debug Console Window */
+  openConsoleWindow() {
+    const dialogRef = this.dialog.open(ConsoleModalComponent);
```

Comment:
no need of dialogRef
only  can wokrs ;) `this.dialog.open(ConsoleModalComponent);`